### PR TITLE
build macos and windows pypy wheels

### PR DESCRIPTION
**Pull request by reaperhulk**

### CODE REVIEW

File: .github/workflows/wheel-builder.yml [hunk 1]
```diff
@@ -24,7 +25,11 @@ jobs:
     steps:
       - run: ${{ matrix.PYTHON.PATH }} -m venv .venv
       - name: Install Python dependencies
-        run: .venv/bin/pip install -U pip wheel cffi setuptools-rust
+        run: .venv/bin/pip install -U pip wheel setuptools-rust
+      - name: Install cffi
+        run: .venv/bin/pip install -U cffi
+        if: contains(matrix.PYTHON.VERSION, 'pypy') == false
```
Comment (alex) [inline .github/workflows/wheel-builder.yml:31]: Why is this required?

File: .github/workflows/wheel-builder.yml [hunk 2]
```diff
@@ -121,6 +137,10 @@ jobs:
           - {ARCH: 'x64', WINDOWS: 'win64', RUST_TRIPLE: 'x86_64-pc-windows-msvc'}
         PYTHON:
           - {VERSION: "3.8", MSVC_VERSION: "2019", "ABI_VERSION": "cp36"}
+          - {VERSION: "pypy-3.7", MSVC_VERSION: "2019"}
+        exclude:
```
Comment (alex) [inline .github/workflows/wheel-builder.yml:141]: Comment explaining why

File: .github/workflows/wheel-builder.yml (hunk 1 above)
Comment (reaperhulk) [reply .github/workflows/wheel-builder.yml:31]: It may not be but cffi is still built into pypy so installing another version is, at best, a no op right?

Comment (alex) [reply .github/workflows/wheel-builder.yml:31]: Yes, but we were producing pypy wheels already here :-)

Comment (reaperhulk) [reply .github/workflows/wheel-builder.yml:31]: Just because it was working doesn't make it correct 😄. I can revert this if you prefer, but unless things have changed with cffi (which maybe it has!) installing cffi on pypy historically could potentially break the built-in cffi.

Comment (alex) [reply .github/workflows/wheel-builder.yml:31]: I have a soft preference for reverting, on readability grounds.

**Review by alex — APPROVED** [review]
